public/app/plugins/datasource/cloudwatch/types.ts

```diff
@@ -73,6 +73,7 @@ export interface CloudWatchJsonData extends AwsAuthDataSourceJsonData {
   database?: string;
   customMetricsNamespaces?: string;
   endpoint?: string;
+  logsTimeout?: string;
```

Comment:
Since it's a string it would be nice with a comment describing the the expected timeout format